packages/shared/src/utils/url/url.js

```diff
@@ -182,6 +182,11 @@ export const getStaticUrl = (
 
     if (is_document) return `${host}/${normalizePath(path)}`;
 
+    // Deriv.com supports languages separated by '-' not '_'
+    if (host === deriv_urls.DERIV_COM_PRODUCTION && lang.includes('_')) {
+        lang = lang.replace('_', '-');
```

Comment:
Is there any particular reason why some languages are separated by `_` and not `-` in the first place?

Reply:
BE and Deriv app is handling languages by '_' while Deriv com uses '-' to support Google SEO